cellprofiler/modules/gammacorrection.py

```diff
@@ -11,7 +11,7 @@
 
 
 class GammaCorrection(cellprofiler.module.ImageProcessing):
-    module_name = "GammaCorrection"
+    module_name = "Advanced"
```

Comment:
The module name should be `GammaCorrection`. You can define the category with `category = "Advanced"`